boards/arm/nrf51_ble400/doc/nrf51_ble400.rst

```diff
@@ -0,0 +1,248 @@
+
+BLE400 is a mother board designed for BLE4.0 Bluetooth 2.4G Wireless Module. Used together with Core board.
+
+Supported Features
+==================
+
+Mother board
+------------
+
+- Onboard battery holder, multi power supplies are available
+- All the IOs are accessible for easy expansion
+- Integrates CP2102 for debugging
+- LEDs and user keys, compatible with other official boards, easy to use
+
+Core board
+----------
+
+- Onboard chip: nRF51822
+- Communication distance (open outdoor 1M data rate): 30m
+- Frequency range: 2.4GHz
+- Operating voltage: 2.0V ~ 3.6V
+- Operating temperature: -40℃ ~ 85℃
+- Expansion pinheader: all the I/Os except P0.26 and P0.27
+- Pinheader pitch: 2.00mm
+- Antenna: onboard antenna
+
+Features
+========
+
+- 2.4 GHz multiprotocol RF transceiver
+- ARM® Cortex™-M0 32 bit processor
+- 128 bit AES HW encryption
+- 256kB flash & 16kB RAM
+- Programmable Peripheral Interconnect (PPI)
+- Digital interfaces: SPI, I2C, UART
+- 10 bit ADC
+- Programmable output power: -20 to +4 dBm
+- Independent application development and protocol stack
+- Fully compatible with NRF24L series
+- Pinout compatible with NRF51xxx series
+- Global separate power management
+- Operating voltage: 1.8 V ~ 3.6 V
+
+Revision History
+================
+
+Core board
+----------
+
+2015.06, Core51822 upgrades the onboard chip to Rev3, features 32kB RAM, supports higher version SDK, and all the demo codes are still compatible with the Rev2 ones.
+
+Connections and IOs
+====================
+
+Mother board
+------------
+
+.. image:: img/BLE400-intro.jpg
+   :width: 500px
+   :align: center
+   :alt: BLE400
+
+1. Wireless module connector: for easily connecting modules like Core51822
+2. MCU pins expansion connector
+3. I2C interface
+4. SPI interface
+5. 5V/3.3V power input/output: usually used as power output, also common-grounding with other user board
+6. USB connector: USB TO UART via onboard convertor CP2102
+7. Debugging interface
+8. UART interface
+9. Battery holder
+10. User LEDs
+11. User keys
+12. Reset button
+13. CP2102
+14. USB TO UART jumper
+15. LEDs & Keys jumper
+
+Core board
+----------
+
+.. image:: img/Core51822-pin.jpg
+   :width: 500px
+   :align: center
+   :alt: Core board pinout
+
+BLE400 dimensions
+=================
+
+.. image:: img/BLE400-size.jpg
+   :width: 500px
+   :align: center
+   :alt: BLE400 size
+
+Connections and IOs
+===================
+
+LED
+---
+
+* LED0 (red) = P0.18
+* LED1 (red) = P0.19
+* LED2 (red) = P0.20
+* LED3 (red) = P0.21
+* LED4 (red) = P0.22
+* SUSPEND = SUSPEND
+* RXD1 = P0.9
+* TXD1 = P0.11
+
+Push buttons
+------------
+
+* KEY1 = P0.16
+* KEY2 = P0.17
+* RESET = SWDIO
+
+I2C
+---
+
+* SMBA = P0.02
+* SCL = P0.01
+* SDA = P0.00
+
+SPI
+---
+
+* NSS = P0.30
+* SCK = P0.25
+* MOSI = P0.24
+* MISO = P0.23
+
+USART1
+------
+
+* RX = P0.05
+* TX = P0.06
+* CTS = P0.07
+* RTS = P0.12
+
+UART (USB)
+----------
+
+* RX = P0.9
+* TX = P0.11
+* CTS = P0.8
+* RTS = P0.10
+
+More photos
```

Comment:
I don't think all these board photos are needed in our project docs.  Instead, just refer folks to https://www.waveshare.com/ble400.htm for more details

Reply:
May be stay pinout photos? Just to make it clear what pin is mapped to what.

Reply:
The other photos on the page can stay.  But this section of "More Photos" should go.

Reply:
Removed "More photos" chapter